### PR TITLE
Install haproxy

### PR DESCRIPTION
**Pull request by drc288**
Create the file to deploy the haproxy

### CODE REVIEW

File: fabric_files/install_nginx.py [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
\ No newline at end of file
+    data = run("echo $HOSTNAME")
+    run("sudo apt-get update")
+    run("sudo apt-get -y install nginx")
+    run("sudo service nginx start")
+    run("sudo mkdir -p /var/www/html")
+    run("sudo touch /var/www/html/index.html")
+    sudo("echo 'Holberton School' > /var/www/html/index.html")
+    sudo("sed -i 's|root /usr/share/nginx/html;|root /var/www/html;|g' /etc/nginx/sites-available/default")
+    sudo('sed -i "s|ipv6only=on;|ipv6only=on;\\n\\tadd_header X-Server-By {};|g" /etc/nginx/sites-available/default'.format(data))
```
Comment (drc288) [inline fabric_files/install_nginx.py:19]: Add this line to put the hostname

File: fabric_files/install_nginx.py [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-        run("sudo service nginx start")
-        run("sudo mkdir -p /var/www/html")
-        run("sudo touch /var/www/html/index.html")
-        sudo("sed -i 's|root /usr/share/nginx/html;|root /var/www/html;|g' /etc/nginx/sites-available/default")
-        sudo("echo 'Holberton School' > /var/www/html/index.html")
-        run("sudo service nginx restart")
-    except Exception:
-        pass
\ No newline at end of file
+    data = run("echo $HOSTNAME")
```
Comment (drc288) [inline fabric_files/install_nginx.py:11]: Capture the data to the hostname

File: fabric_files/install_haproxy.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+env.hosts = ["192.168.33.50"]
+
+
+def install_haproxy():
+    """Install Haproxy in the version"""
+    # run("sudo apt-get update -y")
+    run("sudo add-apt-repository ppa:vbernat/haproxy-1.6")
+    run("sudo apt-get update")
+    run("sudo apt-get install haproxy=1.6.\\* -y")
+    sudo("echo 'frontend haproxynode\n\tbind *:80\n\tmode http\n\tdefault_backend backendnodes' >> /etc/haproxy/haproxy.cfg")
```
Comment (drc288) [inline fabric_files/install_haproxy.py:12]: Improve the echo with the sed

**Comment by drc288** [comment]
[Insytall_haproxy.py](https://trello.com/c/2z8Dc9xz/37-insytallhaproxypy)

**Comment by guxal** [comment]
good

**Review by guxal — APPROVED** [review]

**Pull request merged**